framework_tvb/tvb/tests/framework/interfaces/web/controllers/simulator_controller_test.py

```diff
@@ -578,7 +578,7 @@ def test_set_seeg_monitor_params(self):
             self.simulator_controller.context.add_session_stored_simulator(self.session_stored_simulator)
             self.simulator_controller.context.add_burst_config_to_session(BurstConfiguration(self.test_project.id))
             self.simulator_controller.set_monitors(**self.sess_mock._data)
-            self.simulator_controller.set_monitor_params('iEEGViewModel', **self.sess_mock._data)
+            self.simulator_controller.set_monitor_params('Intracerebral / Stereo EEG', **self.sess_mock._data)
```

Comment:
We will have a problem with using the monitors UI names inside the URLs because in the case of names like `Intracerebral / Stereo EEG` we have a '/' which will be considered part of the URL and treated like there are 2 URL params sent here instead of 1. Try it in the GUI to see how it behaves.